media_kit_video/lib/media_kit_video_controls/src/controls/material.dart

```diff
@@ -865,10 +865,12 @@ class _MaterialVideoControlsState extends State<_MaterialVideoControls> {
         splashColor: const Color(0x00000000),
         highlightColor: const Color(0x00000000),
       ),
+
```

Comment:
Please format the file.

Reply:
I ran dart format but this file didnt change, I think its already formatted?